include/triton/common/logging.h

```diff
@@ -76,36 +77,34 @@ class Logger {
  private:
   std::vector<bool> enables_;
   uint32_t vlevel_;
+  static std::mutex mutex_;
```

Comment:
Why `static`? We have only one global `Logger` object where all writes go through it, so local member variable should be sufficient.

Reply:
Updated, please review.